packages/renderer/src/App.svelte

```diff
@@ -128,13 +128,12 @@ window.events?.receive('navigate', (navigationRequest: unknown) => {
         <Route path="/kube/play" breadcrumb="Play Kubernetes YAML">
           <KubePlayYAML />
         </Route>
-
+        <Route path="/image/run/*" breadcrumb="Images">
```

Comment:
```suggestion
        <Route path="/image/run/*" breadcrumb="Run Image">
```

Reply:
The breadcrumb looks incorrect otherwise 
![image](https://github.com/user-attachments/assets/e7e2c997-399d-4159-84bf-52a408931a5a)